etc/hostcert.conf

```diff
@@ -32,9 +32,3 @@
 ###
 ### To use host certificates from a non standard location
 #  *   /etc/grid-security/cernCA/hostcert.pem     /etc/grid-security/cernCA/hostkey.pem
```

Comment:
This file should be removed as a whole; it only made sense for globus

Reply:
Ok, will do. Thanks.

Reply:
Done.